### PR TITLE
Added images, index.html and css files

### PR DESCRIPTION
**Pull request by ticoniq**
1. Index.html & style.css
- Added navigation for both mobile and desktop view using CSS Flexbox
- Added header banner display 
- Added section bo/ container using CSS Flexbox

1. Added image folder and image files

### CODE REVIEW

**Comment by ticoniq** [comment]
Hi @ticoniq,

This is Tochi Isaiah, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

217957948-6d8d9f33-125c-4724-954f-330eed60533c

### Highlights

- Fixed some of the previous reviewer's requests.

### Required Changes ♻️

- Check the comments under the review.

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @ticoniq  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Good use of Github summary review feature ✔️
> 
> ## Required Changes ♻️

- [ ] Please add inline comments to a line of code or group of code in your review, telling the student what changes to be made, why it should be made and how if possible they can be make the chnages. Please try to be as polite as possible and also use emojis. You can check this tutorial on how best to add inline comments [article](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) 😄 

> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ticoniq** [comment]
I don't understand please explain more

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
Hi @ticoniq 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 

## Invalid Code Review Request ❌ ❌ ❌ 

- [ ] You have submitted a project, but you did not make a review at all, I can only see an opened pull request.

- [ ] According to the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md), you are required to **Copy your code from the project that you completed during the Admissions Trial** and then review it yourself.

- [ ] The main goal of the exercise is that you perform a code review, and then a reviewer will make a review based on how well you performed the code review exercise. :blush: 

- [ ] Please kindly follow the list from the article ["How to give a code review?"](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md).

- [ ] You can also check this [video with an example](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by ticoniq — COMMENTED** [review]
Hi @ticoniq,

This is Tochi Isaiah, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

217957948-6d8d9f33-125c-4724-954f-330eed60533c

### Highlights

- Fixed some of the previous reviewer's requests.

### Required Changes ♻️

- Check the comments under the review.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        </li>
+        <li>
+          <a href="#">About</a>
+        </li>
+        <li>
+          <a href="#">Contact</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
```
[the remark this answers is not in the document]
Comment (V-Blaze) [reply index.html:30]: > Move the `nav` tag into the `header` tag for best html practices, it also helps to organize the HTML code in a structured and semantic way, which can make it easier to maintain and update the website in the future.
- [ ] Hello @ticoniq, You did great Using the GitHub inline comment here and also giving a review. :+1: :100: , But I think there are still some improvements you need to make before moving to the next milestone. 
> Instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it?
 - Looking at this point above which was taken from the article on [how to give code review.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) I think this your comment here sounds more like an order instead of a suggestion. I'll suggest you edit this and rephrase it in a more polite way eg: `I think it will be a good idea to Move the nav tag into the header tag for best html practices, it also helps to organize the HTML code in a structured and semantic way, which can make it easier to maintain and update the website in the future.`

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @, 

# Almost there 🟡 

1094a3e620ebea4cd6cf2d0aceaa8f52


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing some changes from the previous reviewers, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
-  Used the Github inline feature✔
- Gave a reason for the changes required💯 🥇 ✔

 ## Required Changes ♻️

- [x] Hello @ticoniq , You did a great job adding a review and also using the `GitHub inline comments feature` . :+1: :+1:  I think there is still little improvement you need to make before moving to the next milestone.
- Following the article on how to give a review, it suggests Using emojis in your review, I'll also suggest you edit your review comment and add a friendly emoji, this will make the review more human-friendly. :+1: :hugs: :+1: 
> You can include emojis and gifs to make your code review more human-friendly
- Also as part of the requirement for this milestone, You need to add at least 2 inline comment reviews. The second review comment you gave is more of a highlight and not a requirement change for the user. I'll suggest you go through the code and add more review comments, Remember to the polite and try to explain the reason behind your suggestions. :smile: :small_airplane: :100: 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ticoniq** [comment]
Hi @V-Blaze Please because i already did every thing i was ask to do

**Comment by ticoniq** [comment]
Hello @V-Blaze already add the inline comment i don't what else you want me to do

**Review by ticoniq — COMMENTED** [review]
Hi @ticoniq,

This is Tochi Isaiah, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

217957948-6d8d9f33-125c-4724-954f-330eed60533c

### Highlights

- Fixed some of the previous reviewer's requests.

### Required Changes ♻️

- Check the comments under the review.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  <header class="header">
+    <div class="container">
+      <div>
+        <h1>Layout using Flexbox</h1>
+        <p>
+          This layout was building durgin the Flexbox Crash Course with Brad Traversy. He is one of the most valuable
+          people on YouTube and Udemy teaching
+        </p>
+      </div>
+      <img src="images/grid.svg">
```
[the remark this answers is not in the document]
Comment (abdulhamiid) [reply index.html:40]: - [ ] You phrased the comments politely 👍
- [ ] You included an explanation, a "why", for the requested change 💯 
- [ ] Your comment is clear and easy to understand ✔️ 

However, according to the project requirements 👇 
image

Please kindly add a minimum of 2 comments

You can consider editing your previous comment so that it sounds polite as suggested by the previous reviewer. 👍 kindly remember that your job here is to help another person improve in a friendly way. ☺ or you add a new comment. 🛠 

Your previous comment 👇 
image

[OPTIONAL] - You can consider using a couple of emojis and/or gifs in the review.

* They make the code review more human-friendly :alien::robot:.

**Comment by ticoniq** [comment]
Hello @abdulhamiid what else do you want me to do

**Comment by abdulhamiid** [comment]
> Hello @abdulhamiid what else do you want me to do

Hi @ticoniq, as I mentioned in the previous review, according to the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md), you need 2 valid reviews that meet all [these criterias](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) in order to pass this assessment. 

So basically you need to add one more review before moving to the next milestone. 🤗💪

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (abdulhamiid) [reply index.html:40]: > Please always try to add the `alt element` text in your ``. You can imagine a scenario where a user's internet is very slow and not able to load the media of the website, in this kind of situation, 👍 🤗 👍 this `alt element` will help the user to understand what the media is all about. 👍 👍

👆

**Review by ticoniq — COMMENTED** [review]
Hi @ticoniq,

This is Tochi Isaiah, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

217957948-6d8d9f33-125c-4724-954f-330eed60533c

### Highlights

- Fixed some of the previous reviewer's requests.

### Required Changes ♻️

- Check the comments under the review.

File: index.html (hunk 1 above)
Comment (ticoniq) [inline index.html:30]: 👏👏👏 I think it will be a good idea to Move the nav tag into the header tag for best html practices, it also helps to organize the HTML code in a structured and semantic way, which can make it easier to maintain and update the website in the future.

**Review by ticoniq — COMMENTED** [review]
Hi @ticoniq,

This is Tochi Isaiah, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

217957948-6d8d9f33-125c-4724-954f-330eed60533c

### Highlights

- Fixed some of the previous reviewer's requests.

### Required Changes ♻️

- Check the comments under the review.

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @ticoniq 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Perform code review
- ✔️ Use GitHub inline comment feature
- ✔️ Explain a reason fo the requested change
- ✔️ Comments are polite

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
Hello @ticoniq 👋,

This is an additional review as a result of the second opinion request.

We couldn't attend to your second request on time because you submitted the wrong link as PR - [this](https://github.com/abdulhamiid)

Great job 👍 for getting your project approved.

Keep rocking 🚀!
